pkg/network/driver/common.go

```diff
@@ -65,7 +65,6 @@ type NetworkHandler interface {
 	LinkAdd(link netlink.Link) error
 	LinkSetLearningOff(link netlink.Link) error
 	ParseAddr(s string) (*netlink.Addr, error)
-	GetHostAndGwAddressesFromCIDR(s string) (string, string, error)
```

Comment:
typo in the commit msg: invovle vs involve

Please also use ':' as the topic separator instead of '-', for consistency with the commit messages already present in the repo.

The same comment applies to all the commit msgs in this PR.